doc/dev/installation.rst

```diff
@@ -423,6 +423,33 @@ To generate html files for the documentation for Catalyst:
 
 The generated files are located in ``doc/_build/html``
 
+Known Issues
+------------
+
+.. tabs::
+
+   .. group-tab:: Linux Debian/Ubuntu
+
+      .. warning::
+        | If you get this error: 
+        | ``cannot find -lstdc++: No such file or directory``
+        | you might need to install a recent version of ``libstdc``. E.g.: 
+        | ``sudo apt install libstdc++-12-dev``.
+
+      .. warning::
+        | Under Ubuntu 24.04, if you get this error:
+        | ``fatal error: 'Python.h' file not found``
+        | you might need to install the Python Dev package:
+        | ``sudo apt install python-dev-is-python3``.
```

Comment:
The formatting in the rendered docs is not displaying the code parts within their own blocks:

![image](https://github.com/user-attachments/assets/097764cf-f849-4d2c-b719-21be2bc9cc29)

For comparison, see the screenshot below:

![image](https://github.com/user-attachments/assets/da4c5ca1-ae05-498c-a998-7a37b51b6cd8)

Reply:
True. Could you please share the link of the RST viewer/renderer you're using? Couldn't find something useful to render tabs. Thanks in advance!

Reply:
I was just on the [webpage built by the CI](https://xanaduai-pennylane--1122.com.readthedocs.build/projects/catalyst/en/1122/dev/installation.html)

Reply:
Fixed now. Can you check, please?

Reply:
I did not know that feature, thanks @glassnotes !

Reply:
Looks great! One can always build the docs locally, but the fact that the CI can just do it automatically is super convenient 😃